test/cryptography/ECDSA.test.js

```diff
@@ -18,11 +18,12 @@ contract('ECDSA', function ([_, anyone]) {
       // eslint-disable-next-line max-len
       const signatureWithoutVersion = '0x5d99b6f7f6d1f73d1a26497f2b1c89b24c0993913f86e9a2d02cd69887d9c94f3c880358579d811b21dd1b7fd9bb01c1d81d10e69f0384e675c32b39643be892';
 
-      context('with 00 as version value', function () {
+      context('with 00 as version value (wrong)', function () {
         it('works', async function () {
```

Comment:
We should change this `it` description to something like 'returns the zero address', instead of adding '(wrong)' to the `context`.